src/js/api/errors.js

```diff
@@ -78,15 +78,23 @@ export const FLASH_MEDIA_ERROR = 214000;
  * @param {code} [ErrorCode] - The error code.
  * @param {sourceError} [Error] - The lower level error, caught by the player, which resulted in this error.
  */
-export class PlayerError extends Error {
+export class PlayerError {
     constructor(message, code, sourceError = null) {
-        // Avoid passing message via super so that we can define an enumerable message property on PlayerError
-        super();
-        this.code = isValidNumber(code) ? code : null;
+        this.code = isValidNumber(code) ? code : 0;
         this.message = message;
         this.sourceError = sourceError;
     }
 
+    get logMessage() {
+        let codeStr = this.code.toString();
+        const prefix = codeStr.slice(0, 3);
+        const suffix = codeStr.slice(-3);
+        if (suffix >= 400 && suffix < 600) {
```

Comment:
We should avoid using gt/lt operators to compare strings to numbers. 

mod should do the trick:
```js
> 623404 % 1000
404
> 623404 - 404
623000
```